packages/ts/react-form/src/index.ts

```diff
@@ -259,3 +265,18 @@ export function useFormPart<M extends AbstractModel>(model: M): UseFormPartResul
     field,
   };
 }
+
+/**
+ * Hook to access an array model part of a form. It provides the same API as `useFormPart`,
+ * but adds an `items` property that allows to iterate over the items in form of an array of models.
+ *
+ * @param model - The array model to access
+ * @returns The array model part of the form
+ */
+export function useFormArrayPart<M extends ArrayModel>(model: M): UseFormArrayPartResult<M> {
+  const binderNode = getBinderNode(model);
+  return {
+    ...getFormPart(binderNode),
+    items: Array.from(model, (item) => item.model as ArrayItemModel<M>),
+  } satisfies UseFormArrayPartResult<M> as any;
```

Comment:
I guess it's possible to remove this cast now since `extends` in the return type is gone
```suggestion
  }
```